algorithms/yiddish.sbl

```diff
@@ -129,6 +130,8 @@ define mark_regions as (
         (
           test (('{Tsadek}{Vov}{Giml}{Nun}' or '{Tsadek}{Vov}{Kuf}{Tes}' or '{Tsadek}{Vov}{Kuf}{Nun}') atlimit)
           or
+          test ('{Giml}{Ayen}{Beys}{Nun}')
+          or
           ( ['{Giml}{Ayen}'] <- 'GE')
```

Comment:
Should there also be an exception for `'{Giml}{Ayen}{Lamed}{Tes}'` here too?

(Maybe that just isn't a situation that occurs in valid words - I'm afraid I know very little about the Yiddish language...)

Reply:
No, but I should have commented it.

Here Yiddish is acting like a Germanic language (which it is mostly, at least from a morphological point of view). The regular past participle starts with the prefix "ge-", as in "ge-zungen" for the verb "zingen" (to sing).
In the vast majority of cases, it's valid to remove this "ge-". However, there are some common exception, including "gebn" (to give) and "gelt" (money). There are certainly some other exceptions that I could add with time.

However, after the verbal particles, you will never find "gelt" as it isn't a verb.
You could have "avek-gebn" (to give away), but never "avek-gelt".

Also, in prefixed verbs, the "ge-" past participle marker comes after the particle, giving, "avek-ge-gebn" for "gave away". So we only include the "gebn" exception in past participles, not the "gelt".

The form "avek-tsu-gebn" is another particularity (possibly it exists in German as well, but I don't speak any German!). The preposition "tsu" (to) is inserted inside a verb with a particle prefix, e.g. "avek-tsu-gebn" (to give away).

Reply:
The form "avek-tsu-gebn" is another particularity (possibly it exists in German as well, but I don't speak any German!)

You are right, it exists: [wegzugeben](https://de.wiktionary.org/wiki/wegzugeben).

Reply:
Thanks - I see #143 adds a comment about this.